.github/workflows/test.yml

```diff
@@ -873,10 +873,41 @@ jobs:
       matrix:
         spec:
           [
+            './portal/cypress/e2e/journeys/*.spec.js',
```

Comment:
Can you add the reason for this change in the PR description please

Reply:
Done :)